core/grpc-haskell-core.cabal

```diff
@@ -73,6 +73,8 @@ library
   build-tools:          c2hs
   default-language:     Haskell2010
   ghc-options:          -Wall -Werror -fwarn-incomplete-patterns -fno-warn-unused-do-bind
+  if impl(ghc >= 9.8)
+    ghc-options:        -Wno-x-partial
```

Comment:
I believe you want  `impl(ghc >= 9.8) && impl(ghc < 9.8.3)` (or maybe `impl(ghc >= 9.8 && ghc < 9.8.3)`? I'm not sure how you cabal's syntax has you write it, but the bug this flag is guarding against had it's fix merged to GHC 9.8.3).

See: https://gitlab.haskell.org/ghc/ghc/-/issues/24322

Reply:
I can't find where the issue was fixed. Also we are running 9.8.4 https://github.com/awakesecurity/gRPC-haskell/actions/runs/15684778314/job/44185081098?pr=170#step:5:61